README.md

```diff
@@ -1 +1,203 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- [Ruby on Rails](https://rubyonrails.org) - A powerful web application framework for the backend API.
+- [React](https://reactjs.org) - A JavaScript library for building user interfaces on the frontend UI.
+- [PostgreSQL](https://www.postgresql.org) - An open-source relational database.
+- [Bundler](https://bundler.io) - A tool for managing Ruby gem dependencies.
+
+<!-- FEATURES -->
+
+### Key Features <a name="key-features"></a>
+
+- 🔐 **Splash Screen**: Begin your resort journey with confidence! The splash screen features secure authentication. Start your exploring with 
+                         a captivating splash screen that invites you to log in or sign up.
+- 🌏 **Explore Resorts**: Discover a curated selection of resorts, each offering unique experiences and amenities.
+- ✔️ **Reserve Your Spot**: Reserve your desired fav resort.
+- 📅 **Manage Reservations**: Effortlessly manage your resort bookings, including viewing details.
+- 🖥️ **User-Friendly Interface**: Enjoy an user-friendly interface.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name="live-demo"></a>
+- Soon.. 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FRONTEND -->
+
+## 🔗 Link for the Frontend UI <a name="frontend"></a>
+
+- [Resort Vista Frontend](https://github.com/meryemsanem/resort-front-end)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- KANBAN -->
+
+## 🔗 Kanban Board <a name="kanban"></a>
+
+- [Kanban Board](https://github.com/meryemsanem/resort-back-end/projects/1)
+- [Initial Kanban Board](https://user-images.githubusercontent.com/82718562/279994279-135a04c3-6cc8-4814-a12b-964653967c82.png)
+- There are 4 member in our group.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project, you need to have the following installed:
+
+- [Ruby on Rails](https://rubyonrails.org) - A powerful web application framework for the backend API.
+- [Ruby](https://www.ruby-lang.org/en/) - The programming language used by Rails.
+- [PostgreSQL](https://www.postgresql.org) - An open-source relational database.
+- [Bundler](https://bundler.io) - A tool for managing Ruby gem dependencies.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```bash
+git clone https://github.com/meryemsanem/resort-back-end.git
+```
+
+### Install
+
+Install this project with:
+
+```bash
+  cd resort-back-end
+  bundle install
+```
+
+it will install the required gemfile for running the project
+
+### Usage
+
+```bash
+  rails credentials:edit
+  rails db:create
+  rails db:migrate
+  rails db:import_data
```

Comment:
- [ ] When I try to import the data as per instructions, this happens:

![image](https://github.com/meryemsanem/resort-back-end/assets/57421823/6ffc0fd6-119f-403f-a440-002cf59cce78)

Kindly, check your validations and be sure all the data in the JSON file is valid, according to what is written on your models.

![image](https://github.com/meryemsanem/resort-back-end/assets/57421823/ff482418-f816-4815-9947-4c66b938211d)


_note:  I'll edit the model file in order to be able to import the data and continue the review._